E-Commerce/readme.md

```diff
@@ -46,13 +46,33 @@ erDiagram
 Write SQL queries to complete the following tasks:
 
 - [ ] List all the products whose name contains the word "socks"
+
+SELECT * FROM products WHERE product_name LIKE '%socks%';
+
 - [ ] List all the products which cost more than 100 showing product id, name, unit price, and supplier id
+
+SELECT * FROM product_availability JOIN products ON product_availability.prod_id = products.id WHERE unit_price > 100;
```

Comment:
It's better to explicitly list the columns you want to select instead of using `*`. i.e. SELECT products.id, products.name, etc.

So even if the columns you get when you use `*` is the column you want now, this can change in the future if someone adds a new column to any of the tables.